packages/database/src/migrations/20210118040638-AddFetpProject-modifies-data.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const countryCodes = ['PG', 'SB'];
+const projectCode = 'fetp';
+const projectName = 'FETP';
+const projectDescription = 'Field Epidemiologist Training Program Graduate Information';
+
+const dashboardGroupName = 'FETP';
+const projectDashboardReports = '{project_details}';
+const userGroups = ['Public', 'FETP Graduates'];
```

Comment:
From the ticket's description, I think there is another `userGroup` missing here, which is `FETP Admin`?

Reply:
Looks like only 'Public', 'FETP Graduates' userGroups are mentioned in current cards, so we can add a Admin group when required.